pcp/Platform.c

```diff
@@ -461,18 +467,21 @@ void Platform_init(void) {
 
    for (unsigned int i = 0; i < PCP_METRIC_COUNT; i++)
       Platform_addMetric(i, Platform_metricNames[i]);
-   pcp->meters.offset = PCP_METRIC_COUNT;
 
+   pcp->meters.offset = PCP_METRIC_COUNT;
```

Comment:
Why change this grouping of statements? Previously this was aligned with the loop that initializes this (IMHO better) vs. as prelude to the `PCPDynamicMeters_init` call (IMHO less optimal).

Reply:
reverted.